less/carousel-angled.less

```diff
@@ -0,0 +1,166 @@
+.carousel-angled {
+  .carousel-inner {
+    @media (min-width: @screen-lg) {
+      height: 76vh;
+      min-height: 580px;
+      max-height: 1096px;
+    }
+
+    .item {
+      @media (min-width: @screen-lg) {
+        height: 100%;
+
+        &.active {
+          .carousel-caption {
+            opacity: 1;
+            transition: 0.3s 0.1s;
+          }
+        }
+
+        &.left,
+        &.right {
+          .carousel-caption {
+            opacity: 0;
+            transition: 0.1s;
+          }
+        }
+      }
+
+      > img {
+        width: 100%;
+        height: 36vh;
+        object-fit: cover;
+        @media (min-width: @screen-lg) {
+          position: absolute;
+          height: inherit;
+          margin-top: (@spacer * 12);
+        }
+      }
+    }
+  }
+
+  .carousel-caption {
+    display: flex;
+    position: relative;
+    bottom: inherit;
+    left: 0;
+    align-items: center;
+    width: 100%;
+    height: 100%;
+    min-height: (@spacer * 25);
+    padding-top: (@spacer * 4);
+    background-color: @brand-white;
+    color: @text-color;
+    text-align: center;
+    text-shadow: none;
+    @media (min-width: @screen-lg) {
+      top: (@spacer * 12);
+      height: 100%;
+      padding: 0;
+      transform: translateY((-@spacer * 8));
+      background: none;
+      text-align: left;
+      opacity: 0;
+    }
+
+    &:before {
+      @media (min-width: @screen-lg) {
+        position: absolute;
+        top: 0;
+        left: -55%;
+        width: 100%;
+        height: 85%;
+        transform: skew(-23deg);
+        background-color: @brand-white;
+        box-shadow: 0 20px 66px 0 rgba(34,48,73,.2);
+        content: "";
+      }
+      @media (min-width: @screen-xl) {
+        left: -80%;
+        width: 130%;
+      }
+    }
+
+    h1,
+    h2,
+    h3,
+    h4,
+    h5,
+    h6 {
```

Comment:
Default carousel.less have this styles:
```
  h1, h2, h3, h4, h5, h6 {
      color: @carousel-caption-color;
  }
````
These ones in the angled version change the default white colour. They're not necessary but they're useful since the background of the caption is also white.